pod-operation/hx_wheel_encodeer

```diff
@@ -0,0 +1,39 @@
+use std::thread::sleep;
+use std::time::Duration;
+use rppal::gpio::{Gpio, Level};
+
+const OUTPUT_A: u8 = 6;
+const OUTPUT_B: u8 = 7;
+
+fn main() {
+    let mut counter = 0;
+    let mut a_state;
+    let mut a_last_state;
+
+    let gpio = Gpio::new().unwrap();
+    let mut output_a = gpio.get(OUTPUT_A).unwrap().into_input();
+    let mut output_b = gpio.get(OUTPUT_B).unwrap().into_input();
+
+    // Reads the initial state of OUTPUT_A
+    a_last_state = output_a.read().unwrap();
+
+    loop {
+        a_state = output_a.read().unwrap(); // Reads the "current" state of OUTPUT_A
+
+        // If the previous and the current state of OUTPUT_A are different, that means a Pulse has occurred
+        if a_state != a_last_state {
+            // If the OUTPUT_B state is different from the OUTPUT_A state, that means the encoder is rotating clockwise
+            if output_b.read().unwrap() != a_state {
```

Comment:
Issue: tracking changes only on one of the values will reduce the resolution by half and does not have any fault tolerance and can lead to incorrectly interpreting the direction. See [last year's wheel_encoder.py](https://github.com/UCI-HyperXite/hx8-control-system/blob/main/pod-control/src/components/wheel_encoder.py) for how to track both values (A and B) on every increment.

Reply:
Noted! I’ll resolve this by EOW